Naming.java

```diff
@@ -0,0 +1,31 @@
+package az.alizeynalli.cleancode.naming;
+
+public class Naming {
+
+    public Date d;
+    public List<Testcase> find(User user){};
+    
+    public Account[] AccountList;
+
+    public void deleteUser(User pUser){};
+    
+    public URL r;
```

Comment:
// Avoid mental mapping
    // not everybody would think r will be url
    // Good  
    public URL url;